src/libmongoc/tests/test-mongoc-bulk.c

```diff
@@ -2984,26 +2984,26 @@ test_unordered_bulk_writes_with_error (void)
    collection = mongoc_client_get_collection (client, "db", "test");
    /* use an unordered bulk write; we expect to continue on error */
    bulk = mongoc_collection_create_bulk_operation_with_opts (
-      collection, tmp_bson("{'ordered': false}"));
+      collection, tmp_bson ("{'ordered': false}"));
```

Comment:
clang-format fixed some unrelated code, which I didn't bother to exclude from the diff.